package.json

```diff
@@ -76,7 +76,7 @@
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^4.6.0",
     "ember-resolver": "^8.0.0",
-    "ember-source": "~3.25.0",
+    "ember-source": "~3.27.0-alpha.2",
```

Comment:
I've upgraded here and also added the `ember` key below just so I can test locally. I'll remove these before merging